lib/rspec/rails/matchers.rb

```diff
@@ -22,4 +22,5 @@ module Matchers
 require 'rspec/rails/matchers/have_http_status'
 if RSpec::Rails::FeatureCheck.has_active_job?
   require 'rspec/rails/matchers/active_job'
+  require 'rspec/rails/matchers/have_enqueued_mail'
```

Comment:
Don't we need a test?

Reply:
Hi Benoit, thanks for the review!

I added a cucumber feature for `have_enqueued_mail` matcher, which would fail without this `require` line..